app/Http/Controllers/RemoteAttendanceController.php

```diff
@@ -41,17 +72,43 @@ public function index(Request $request, string $secret)
             $att->attendable_type = $attendable_type;
             $att->attendable_id = $attendable_id;
             $att->gtid = $gtid;
-            $att->source = 'secret-link';
+            $att->source = 'secret-link-'.$link->id;
```

Comment:
Should we maybe put the link ID in a separate column?

Reply:
Agreed and implemented.